examples/idos-example-dapp/src/main.js

```diff
@@ -286,21 +286,27 @@ const connectWallet = {
         throw new Error("Unreachable");
     }
 
+    const page = 1;
+    const size = 10;
     const { grants } = await terminal
       .h1("eyes", "User's grants to this dApp")
-      .wait("awaiting RPC", cache.get("grants") || idos.grants.getGrantsOwned());
+      .wait("awaiting RPC", idos.grants.getGrantsGranted(page, size));
```

Comment:
Two issues:
- Why did we stop using a cache?
- Pretty sure we want to get all entries here. If not, we should add some UI element making that obvious, maybe? The only thing I'm concerned is about creating the wrong mental model on whomever is using this. I'd be ok with displaying the total count of grants, and then show the first 10.